zdigness/puttdle
# [TASK] Correct Docker Compose

## Description
Docker compose was added to the entire project in https://github.com/zdigness/puttdle/pull/22. The old env config was setup, however we needed to correct that since the `.env` file isn't in the same dir as the `docker-compose` file anymore.

## Motivation and Context
1. Correct [env var](https://docs.docker.com/compose/environment-variables/set-environment-variables/) for docker-compose for `server` container
2. Update docker-compose to remove `version` since it's [not included anymore](https://docs.docker.com/compose/compose-file/legacy-versions/)
3. Remove `.github/workflows/docker-compose.yml` since `validate-docker-compose.yml` already existed

## How Has This Been Tested?
1. Go into root dir
2. Ensure the `.env` file has been created in the `./server/` dir
4. Run `docker-compose up`
5. See console

## Artifacts (if appropriate):

https://github.com/zdigness/puttdle/assets/23369330/5996e0ae-8d92-45bb-902d-ee8d03e735e5
